packages/validation/src/sarif/suppressions/AnnotationSuppressor.ts

```diff
@@ -0,0 +1,36 @@
+import {RuleMetadata, Suppression, ValidationResult} from '../../common/sarif';
```

Comment:
This file looks like leftover. Probably replaced by packages/validation/src/sarif/suppressions/plugins/AnnotationSuppressor.ts?